RELEASE_PROCESS.md

```diff
@@ -297,3 +317,13 @@ spelled with complete minutiae.
       1. A blue box will appear above the job list with a right side drop
          -down. Click the drop-down and specify the version number in the
          dialog that appears
+
+1. Annouce the release
+      1. For major and minor releases, write a blog and publish it to blog.podman.io
+         Highlight key features and important changes or fixes. Link to the github release.
+         Make sure the blog is properly  tagged with the Announcement, Release, and Podman tags,
+         and any other appropriate tags.
+      1. Send an email to the podman and podman-desktop mailing lists.
+         Link the to release blog and github release.
+      1. Tweet the release.
```

Comment:
We have a Mastodon account, too (https://fosstodon.org/home). It should also be posted there as well.